tests/src/ComparisonUtils.h

```diff
@@ -9,13 +11,22 @@
 #include "UTIL/LCIterator.h"
 #include "EVENT/LCCollection.h"
 
+#include "podio/RelationRange.h"
+#include "podio/ObjectID.h"
+
 #include <cmath>
 #include <array>
 #include <iostream>
 #include <vector>
+#include <algorithm>
+
+inline std::ostream& operator<<(std::ostream& os, const podio::ObjectID id)
+{
+  return os << "[" << id.collectionID << ": " << id.index << "]";
+}
 
```

Comment:
```suggestion
```
This leads to ambiguous resolution since podio::ObjectID already has an `operator<<`